### PR TITLE
Hey guys updated the help request form to include the medic notes 

### PR DESCRIPTION
**Pull request by Faith-qa**

### CODE REVIEW

File: helprequest/views.py [hunk 1]
```diff
@@ -47,9 +47,22 @@ class RequestHelp(FormView):
     """
     TODO: on save of the help request, link the user requesting help to the patient fild of the help request
     """
+
+
     template_name = 'help_request.html'
     form_class = HelpRequestForm
     success_url = 'status'
+    def gethelp(request):
```
Comment (Emmastro) [inline helprequest/views.py]: Where is this function being called/used?

File: helprequest/views.py [hunk 2]
```diff
@@ -73,3 +86,20 @@ def form_valid(self, form):
...
+    model = HelpRequest
+    fields = [
+        # "slug"
+        "medic_notes"
+    ]
+    template_name = 'help_response.html'
+    success_url = '/'
+        
+
+def form_valid(self, form):
```
Comment (Emmastro) [inline helprequest/views.py:90]: Where is this function being called/used? Also, is it inside the HelpRequestUpdate class or outside - it looks the indentation is wrong :) 
Also, if you're not adding any functionality to the function, there is no need to override it

File: helprequest/migrations/0003_helprequest_slug.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.4 on 2021-07-07 03:48
+
+import autoslug.fields
```
Comment (Emmastro) [inline helprequest/migrations/0003_helprequest_slug.py:3]: Is this a new requirement? If yes, can you add it on the requirements.txt, and update the README as of why we need this and how we're using it?

**Review by Emmastro — CHANGES_REQUESTED** [review]

File: helprequest/migrations/0003_helprequest_slug.py (hunk 3 above)
Comment (Faith-qa) [reply helprequest/migrations/0003_helprequest_slug.py:3]: hey...so I need to delete it... was trying to use it to make the URL work [update view] but it kind of failed...it's no longer a requirement

File: helprequest/views.py (hunk 1 above)
Comment (Faith-qa) [reply helprequest/views.py]: yeah just deleted it...it's not being used elsewhere

**Review by Peter5793 — COMMENTED** [review]
The changes are valid,  was the change in the form_valid placed?

**Pull request merged**